layouts/partials/tw/nav.html

```diff
@@ -1,137 +1,268 @@
-<nav
+<div
   id="top"
   x-data="{ isOpen: false }"
-  class="site-nav"
```

Comment:
Let's delete the old .site-nav classes from tw.css.

Reply:
Updated: https://github.com/spotlightpa/poor-richard/pull/422/commits/9e0d40451ab400ba67268e0e18907b7a28bc9df6